src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -1049,7 +1088,200 @@ namespace HVACVariableRefrigerantFlow {
 		Real64 const Te, // Outdoor unit evaporating temperature
 		Array1< Real64 > const & Par // Array of parameters
 	);
-
+	
+	void
+	VRFOU_TeTc(
+		int const VRFCond, // Index to VRF outdoor unit
```

Comment:
@Myoldmopar It's a good idea to start the VRF OO design from these new functions. I will refactor them soon.